src/main_prog.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+
+import material_properties
+
+import generate_mesh
+
+# Allow for user input for number of materials and what kind of materials
+
+parser = argparse.ArgumentParser()
+    
+parser.add_argument("-n", "--number_materials", type=int, default=3, required=False,
+                           help="Determines how many materials are in the particle")
+
+
+parser.add_argument("-f", "--fuel", type=list(tuple), default=[('UO2',250e-6)], required=False,
+	               choices=[('UO2',250e-6),('UF4',250e-6)],
```

Comment:
Don't you want to allow users to specific different radii?  I can see limiting the choice of material, but the choice of radius is limiting.  Perhaps this can be changed in the future, but you might want to reconsider the data model and whether there is a way to define the allowable choices for this without constraining the radius. It might require two separate arguments.

Reply:
Yes, you are correct. I initially just set arbitrary radii to see if it would work. I am currently working on changing the data model to allow for any input for the radii.